src/Compiler/CompilerPasses.hpp

```diff
@@ -20,7 +20,8 @@ void addONNXToMLIRPasses(mlir::PassManager &pm);
 void addONNXToKrnlPasses(
     mlir::PassManager &pm, int optLevel, bool enableCSE = true);
 void addKrnlToAffinePasses(mlir::PassManager &pm);
-void addKrnlToLLVMPasses(mlir::OpPassManager &pm, bool enableCSE = true);
+void addKrnlToLLVMPasses(mlir::OpPassManager &pm, bool enableCSE = true,
+    bool verifyInputTensors = false);
```

Comment:
nit: it looks like verifyInputTensor is always defined explicitly, so maybe no need to make it with a default value?